scripts/experiments_tabular/maf.jl

```diff
@@ -70,16 +96,26 @@ end
 try_counter = 0
 max_tries = 10*max_seed
 cont_string = (contamination == 0.0) ? "" : "_contamination-$contamination"
+sampling_string = sampling == "bayes" ? "_bayes" : ""
```

Comment:
Seriously, we should put this together as one function in GenerativeAD.jl, I don't see much variance of this block of code between my models and the others I think are more or less the same.